### ISSUE TITLE
Kubernetes: Renovate hasn't opened a PR in over 24 hours

### ISSUE THREAD

**Issue by ggilmore** [renovatebot/renovate#2941]
**What Renovate type are you using?**
Renovate GitHub App

**Describe the bug**
In github.com/sourcegraph/infrastructure, we configured Renovate to update our docker image tags in our Kubernetes manifests. We've just noticed that Renovate hasn't opened a PR in over 24 hours (see [Renovate job log for the repo here](https://renovatebot.com/dashboard#github/sourcegraph/infrastructure)) even though one of the images [(sourcegraph/server)](https://hub.docker.com/r/sourcegraph/server/tags/) has had many recent pushes. 

Are we running into a rate limit of some kind?

**Comment by rarkins** [comment]
Just to confirm: were Kubernetes updates working previously?

**Comment by ggilmore** [comment]
Yes, it was working fine last week. We haven't changed anything in our renovate.json.

**Comment by rarkins** [comment]
Can you provide (or email, to support@renovatebot.com) a copy - redacted if necessary - of one of the manifests that's failing to be updated? i.e. something that parses correctly I can test against locally.

**Comment by ggilmore** [comment]
Here you go:


```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: sourcegraph-server
  annotations:
    description: The single docker-image version of Sourcegraph. 
  labels:
    deploy: sourcegraph-server
spec:
  selector:
    matchLabels:
      app: sourcegraph-server
  strategy:
    type: Recreate
  template:
    metadata:
      labels:
        app: sourcegraph-server
    spec:
      containers:
      - name: sourcegraph-server
        image: sourcegraph/server:insiders@sha256:6f10704ceed754b558e12295cbf7290ebddc0bcc61e1128280eadab58fc95f0a
        env:
        - name: SRC_LOG_LEVEL
          value: dbug
        ports:
        - containerPort: 7080
          name: http
        - containerPort: 7443
          name: https
        resources:
          requests:
            cpu: "6"
            memory: 30G
          limits:
            cpu: "12"
            memory: 60G
        volumeMounts:
          - mountPath: /etc/sourcegraph
            name: site-config
          - mountPath: /var/opt/sourcegraph
            name: data
      volumes:
        - name: site-config
          persistentVolumeClaim:
            claimName: site-config
        - name: data
          persistentVolumeClaim:
            claimName: data

```

**Comment by rarkins** [comment]
Just to clarify, that sha256 seems to be up to date - did you update it manually?

**Comment by ggilmore** [comment]
Ah, sorry. We updated manually after we noticed that Renovate wasn't doing it. Here is an "old" version:

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: sourcegraph-server
  annotations:
    description: The single docker-image version of Sourcegraph. 
  labels:
    deploy: sourcegraph-server
spec:
  selector:
    matchLabels:
      app: sourcegraph-server
  strategy:
    type: Recreate
  template:
    metadata:
      labels:
        app: sourcegraph-server
    spec:
      containers:
      - name: sourcegraph-server
        image: sourcegraph/server:insiders@sha256:c22494819a337718c649e7cf92ff3e4cee29071d3c4ce10b39f007d324c33d42
        env:
        - name: SRC_LOG_LEVEL
          value: dbug
        ports:
        - containerPort: 7080
          name: http
        - containerPort: 7443
          name: https
        resources:
          requests:
            cpu: "6"
            memory: 30G
          limits:
            cpu: "12"
            memory: 60G
        volumeMounts:
          - mountPath: /etc/sourcegraph
            name: site-config
          - mountPath: /var/opt/sourcegraph
            name: data
      volumes:
        - name: site-config
          persistentVolumeClaim:
            claimName: site-config
        - name: data
          persistentVolumeClaim:
            claimName: data

```

**Comment by rarkins** [comment]
I tested the above config and it updated the digest no problem, to the very latest digest.

I tried running the bot in sourcegraph/infrastructure again but it looks like someone already updated to the latest digest? (the value below is newer than what we discussed earlier).

`{"hostname":"ip-172-31-20-114","level":30,"repository":"sourcegraph/infrastructure","depName":"sourcegraph/server","currentValue":"insiders","currentDigest":"sha256:2d9223dd04e584efb33d0dd7d6a46129f5edd3b85a71b2b0f6e0dac0ef937682","msg":"Kubernetes image","time":"2018-12-11T21:36:22.089Z"}`

**Comment by rarkins** [comment]
Summary: 
- failed to reproduce the problem - when run in a test repo with renovate CLI, it updates as expected.
- failed to test in actual repo this evening as it seems there's a human making rapid updates manually

**Comment by rarkins** [comment]
I watched for updates to the sourcegraph/server:insiders image and when I noticed an update I enqueued the sourcegraph/infrastructure repo just now. PR #790 was created as expected, updating the digest.

I don't think anything was wrong before, because there has been no code change to production since you raised this issue. 

For your info:
- Renovate queues up each repo hourly, but can run more often if there is a relevant webhook event such as someone merging a Renovate PR
- Docker digest values are cached for 30 minutes, i.e. we don't requery the same tag more frequently than once per 30 minutes. Perhaps we could lower this if you think it's a problem.
